src/main/java/core/basesyntax/servise/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.servise;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REPORT_HEADER = "fruit,quantity" + System.lineSeparator();
```

Comment:
To follow the encapsulation principle, consider making `REPORT_HEADER` private if it is not intended to be used outside this class.